### PR TITLE
Implement aanschrijfwijze feature

### LINKED ISSUE #164: Implement aanschrijwijze feature
Implement feature described here: https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/blob/v1.0.0/features/aanschrijfwijze.feature

### PR DESCRIPTION
**Pull request by shea-maykinmedia**
Fixes #164 

Implements the scenarios described here https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/blob/v1.0.0/features/aanschrijfwijze.feature

I branched off of 'implement-aanhef-feature' so once that pull request is merged then I can merge this one into master.

### CODE REVIEW

File: src/openpersonen/features/aanschrijfwijze/test/test_aanschrijfwijze.py [hunk 1]
```diff
@@ -0,0 +1,505 @@
...
+                voornamen,
+                voervoegsel_partner,
+                geslachtsnaam_partner,
+                aanduiding_aanschrijving,
+                None,
+                None,
+                None,
+            )
+
+            self.assertEqual(aanschrijfwijze, result)
```
Comment (stevenbal) [inline src/openpersonen/features/aanschrijfwijze/test/test_aanschrijfwijze.py:47]: Not really necessary, but I like to use `subTest` to get some more info if an assertion fails on an iteration

```suggestion
        for row in table_rows:
            (
                aanduiding_aanschrijving,
                samenstelling,
                voervoegsel,
                geslachtsnaam,
                voornamen,
                voervoegsel_partner,
                geslachtsnaam_partner,
                aanschrijfwijze,
            ) = row
            with self.subTest(samenstelling=samenstelling):
                result = get_aanschrijfwijze(
                        voorvoegsel,
                        geslachtsnaam,
                        voornamen,
                        voervoegsel_partner,
                        geslachtsnaam_partner,
                        aanduiding_aanschrijving,
                        None,
                        None,
                        None,
		)
		
                self.assertEqual(aanschrijfwijze, result)
```

**Review by stevenbal — APPROVED** [review]
Looks good overall :+1:

File: src/openpersonen/features/aanschrijfwijze/test/test_aanschrijfwijze.py (hunk 1 above)
Comment (shea-maykinmedia) [reply src/openpersonen/features/aanschrijfwijze/test/test_aanschrijfwijze.py:47]: Ya, I think subTest would be good to use for these.

File: src/openpersonen/features/aanschrijfwijze/aanschrijfwijze.py [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+
+
+def get_aanschrijfwijze_first_name(first_name):
+    split_first_name = first_name.split(" ")
+    aanschrijfwijze_first_name = ""
+
+    for name in split_first_name:
+        aanschrijfwijze_first_name += f"{name[0]}."
+
+    return aanschrijfwijze_first_name
```
[the remark this answers is not in the document]
Comment (shea-maykinmedia) [reply src/openpersonen/features/aanschrijfwijze/aanschrijfwijze.py:11]: 👍

**Pull request merged**